src/NuGet.Services.Metadata.Catalog.Monitoring/Notification/LoggerMonitoringNotificationService.cs

```diff
@@ -45,11 +48,15 @@ public void OnPackageValidationFinished(PackageValidationResult result)
                     }
                 }
             }
+
+            return Task.FromResult(0);
         }
 
-        public void OnPackageValidationFailed(string packageId, string packageVersion, IList<JObject> catalogEntriesJson, Exception e)
+        public Task OnPackageValidationFailed(string packageId, string packageVersion, IList<JObject> catalogEntriesJson, Exception e, CancellationToken token)
         {
             _logger.LogError(LogEvents.ValidationFailedToRun, e, "Failed to test {PackageId} {PackageVersion}! {ExceptionMessage}", packageId, packageVersion, e.Message);
```

Comment:
Why do you log the `ExceptionMessage` if you also log the exception in the second parameter?